### DISCUSSION TITLE
Commits does not updated on github page

### DISCUSSION THREAD

**Discussion by Asonjay** [alshedivat/al-folio#2267] [Q&A]
Hi, 

I have encountered an issue that all of my commits recently does not deployed to the Github page. The code has been updated to master branch, but it did not render on the Github page.

Thanks.

**Comment by george-gca** [comment]
Is it working locally?

**Reply by Asonjay** [reply]
Yes.

**Reply by Asonjay** [reply]
The reason is that I removed prettier.yml from the workflow. After the removal, it will take a long time for Github to deploy my code. Now, all the commits have been updated.

**Reply by george-gca** [reply]
You can check the actions tab on your repository to check the status of the workflows.
